config/routes.rb

```diff
@@ -37,4 +37,9 @@
     post 'update', to: 'rescue_requests#update_short', as: :request_update
     post 'submit', to: 'rescue_requests#update_long', as: :request_submit
   end
+
+  scope '/api' do
+    get 'geojson', to: 'api#geojson', as: :api_geojson
```

Comment:
To leave us room for future expansion, I'd put both these routes in a `scope 'requests'` block - so the final route would be `/api/requests/geojson`. Then if we want a GeoJSON API for another data type, we can do it.

Reply:
Fixed in the lastest commit.